src/logic/player.cc

```diff
@@ -92,6 +93,10 @@ void terraform_for_building(Widelands::EditorGameBase& egbase,
 
 namespace Widelands {
 
+Player::Field::PFB::PFB() {
```

Comment:
What does PFB mean?

Reply:
`PartiallyFinishedBuilding`. This abbreviation is used in a few other places as well. I would have preferred a nameless `union` here since it is used nowhere else but the compiler demanded an explicit constructor.

Reply:
Well, if I have to ask, the name is not clear. Can you rename all of them to `PartiallyFinishedBuilding`please?